packages/opentelemetry-plugin-grpc/src/grpc.ts

```diff
@@ -331,14 +331,14 @@ export class GrpcPlugin extends BasePlugin<grpc> {
   ): string[] {
     const methodsToWrap = [
       ...Object.keys(methods),
-      ...(Object.keys(methods)
+      ...Object.keys(methods)
         .map(methodName => methods[methodName].originalName)
         .filter(
           originalName =>
             // eslint-disable-next-line no-prototype-builtins
             !!originalName && client.prototype.hasOwnProperty(originalName)
-        ) as string[]),
-    ];
+        ),
+    ].filter((name, index, list) => list.indexOf(name) === index) as string[]; // remove duplicates
```

Comment:
Would probably be easier and possibly more performant to handle this with a `Set`

```typescript
return Array.from(
  new Set([
    ...Object.keys(methods),
    ...Object.entries(methods)
      .map([name, originalName] => originalName)
      .filter(originalName =>
        !!originalName && client.prototype.hasOwnProperty(originalName)
      ),
  ])
);
```

or

```typescript
const methodSet: Set<string> = new Set();
for (const [name, { originalName }] of Object.entries(methods)) {
  methodSet.add(name);
  if (originalName && client.prototype.hasOwnProperty(originalName)) {
    methodSet.add(originalName);
  }
}
return Array.from(methodSet);
```